### DISCUSSION TITLE
Cannot enable git credential manager when installing Git for Windows

### DISCUSSION THREAD

**Discussion by ronaldwkchan** [community/community#22137] [New to GitHub]
<p><img src="https://user-images.githubusercontent.com/53422560/181069018-ae02664c-60b5-467e-9784-7c8be71bca6a.png" alt="credential.PNG" data-base62-sha1="qkpgGFEZOQnpsZMDQr5AMMAO5c0" width="585" height="477"></p>
<p>Option ‘Enable Git Credential Manager’ is dim when installing Git for Windows. Cannot select the option. What is the possible reason?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/ronaldwkchan">@ronaldwkchan</a>! :wave:</p>
<p>Did you ever manage to solve this?</p>
<p>I think the first thing I would try is to install Git Credential Manager for Windows separately, so make sure you have the most up to date version of it, and then try installing Git Bash for Windows again!</p>
<p>Instructions for installing the Credential Manager can be found here:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/Microsoft/Git-Credential-Manager-for-Windows#download-and-install" target="_blank">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/19803303/181069020-24b2557c-1c1d-4c28-861d-38a830d1a9ce.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/Microsoft/Git-Credential-Manager-for-Windows#download-and-install" target="_blank">microsoft/Git-Credential-Manager-for-Windows</a></h3>

<p>Secure Git credential storage for Windows with support for Visual Studio Team Services, GitHub, and Bitbucket multi-factor authentication. - microsoft/Git-Credential-Manager-for-Windows</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
